server/src/test/java/org/elasticsearch/index/mapper/BooleanScriptFieldTypeTests.java

```diff
@@ -319,7 +319,7 @@ protected Query randomTermsQuery(MappedFieldType ft, SearchExecutionContext ctx)
     }
 
     public void testDualingQueries() throws IOException {
-        BooleanFieldMapper ootb = new BooleanFieldMapper.Builder("foo", ScriptCompiler.NONE, Version.CURRENT).build(
+        BooleanFieldMapper ootb = new BooleanFieldMapper.Builder("foo", ScriptCompiler.NONE,false, Version.CURRENT).build(
```

Comment:
There is some code formatting problem here that you can detect by running `./gradlew :server:spotlessCheck`. 
The easiest way to get rid of these is to run `./gradlew :server:spotlessApply` (or `./gradlew spotlessApply to check all project submodules) before commit. I also tend to forget that but our build will not pass otherwise ;-)

Reply:
Sorry for such a silly thing. I should have checked these beforehand. This has been fixed.